src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/SprintOnBacklog.kt

```diff
@@ -0,0 +1,30 @@
+package com.atlassian.performance.tools.jirasoftwareactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+import java.time.Duration
+
+class SprintOnBacklog(
+    private val driver: WebDriver,
+    val name: String
+) {
```

Comment:
Let's keep page object details in the page object:
```suggestion
class SprintOnBacklog(
    private val driver: WebDriver,
    private val element: WebElement
) {
    val name: String
        get() = element.findElement(By.className("ghx-name")).text.findElement(By.className("ghx-name")).text
```